webapp_server/chart/cosmotech-business-webapp/values.yaml

```diff
@@ -0,0 +1,30 @@
+# name is the name of the deployment, ingress and service that will be created
+name: cosmotech-business-webapp
+# imageName is the name of the docker image containing your webapp server (e.g. ghcr.io/cosmo-tech/azure-sample-webapp)
+imageName: ""
+# imageTag is the tag of the docker image to pull
+imageTag: latest
+# functionsImageName is the name of the docker image containing your API functions server
+functionsImageName: ""
+# functionsImageTag is the tag to pull for functionsImageName
+functionsImageTag: latest
+# keycloakRealmUrl is the URL of the keycloak realm used to authenticate users of the webapp (necessary to valide user tokens in the GetEmbedInfo function)
```

Comment:
In this type of PRs, I can only be useful in reading comments :sweat_smile: probably a typo : to _validate_ :slightly_smiling_face:

Reply:
Nice catch, thank you ! :smile: